src/averageAges.js

```diff
@@ -14,12 +15,27 @@
  *
  * @return {number}
  */
+
+const SEX_MALE = 'm';
+const SEX_FEMALE = 'f';
+const ONE_CENTURY = 100;
+
+function reduceAverage(array) {
+  return array.reduce((personAge, person) => {
+    return personAge + (person.died - person.born) / array.length;
+  }, 0);
+}
```

Comment:
Do not use too general naming and break complex parts into simpler ones.
```suggestion
function getAverageAge(people) {
  const totalAge = people.reduce((total, { died, born }) => total + (died - born), 0);
  return totalAge / people.length;
}
```